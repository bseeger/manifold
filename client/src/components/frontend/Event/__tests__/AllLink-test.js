import React from 'react';
import renderer from 'react-test-renderer';
import AllLink from '../AllLink';

describe("Frontend.Event.AllLink Component", () => {

  it('renders correctly', () => {
    const component = renderer.create(
      <AllLink
        count={5}
        threshold={3}
        projectId="1"
      />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
