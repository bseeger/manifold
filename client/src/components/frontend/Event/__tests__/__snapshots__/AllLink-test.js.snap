exports[`Frontend.Event.AllLink Component renders correctly 1`] = `
<div
  className="section-heading-utility-right">
  <a
    className="button-primary"
    onClick={[Function]}
    style={Object {}}>
    See all Activity
  </a>
</div>
`;
